pynxtools/dataconverter/readers/xps/reader.py

```diff
@@ -311,6 +311,21 @@ def fill_from_value(key):
             fill_from_value(key)
 
 
+def _concatenate_values(value1, value2):
```

Comment:
This function allows parsing of more than one raw data file. In the read method of XPSReader, the data dicts of the two files are merged. On keys that are in both dictionaries (especially on 'data'), the values are merged, depending on the type of the values: dicts (e.g. in data) are merged, strings are stored in lists, and lists are concatenated.